docs/misc/making-docs-guide.md

```diff
@@ -0,0 +1,49 @@
+---
+title: How to contribute to DataHub docs?
+date: 2017-12-21
+authors: ['Mikanebu']
+---
+
+
+### How to contribute to DataHub docs?
+
+You are always welcome to open PR for DataHub docs.
+Since you know how markdown works, there are only couple steps you need to know:
+
+* Template - template for home docs page is located in `views/docs_home.html`, the rest docs use `views/docs.html`
```

Comment:
Do we really need all of this? I thought it is something so users can contribute to existing docs not create new ones